requirements.txt

```diff
@@ -0,0 +1,8 @@
+cycler==0.10.0
+env==0.1.0
```

Comment:
I created a small env.py file, I don't think this is a real package....

Reply:
Yeah I saw that, but I wasn't 100% sure because the env package exists and the env.py is located in test, which would need from .test import env to import, no ? I'l be looking into it.

Reply:
Ok, I will clearly need to change its name.

Reply:
Done.